### PR TITLE
adding terraform code

### PR DESCRIPTION
**Pull request by suneel00**
Hi @SPadhi-dev ,

Can you review terraform code and approve my PR.

Regards,
Suneel

### CODE REVIEW

File: terraformcode.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+data "aws_ami" "ubuntu" {
+  most_recent = true
+
+  filter {
+    name   = "name"
```
Comment (SPadhi-dev) [inline terraformcode.txt:5]: Change the name as jenkins server

File: terraformcode.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["099720109477"] # Canonical
+}
+
+resource "aws_instance" "web" {
+  ami           = data.aws_ami.ubuntu.id
+  instance_type = "t3.micro"
```
Comment (SPadhi-dev) [inline terraformcode.txt:19]: change instance state as t2 arge

**Review by SPadhi-dev — COMMENTED** [review]
Please work on the view commands

File: terraformcode.txt (hunk 1 above)
Comment (suneel00) [reply terraformcode.txt:5]: completed

File: terraformcode.txt (hunk 2 above)
Comment (suneel00) [reply terraformcode.txt:19]: changed

**Comment by suneel00** [comment]
can u please review code

**Review by SPadhi-dev — APPROVED** [review]
looks good

**Pull request merged**